app/Http/Controllers/NewsController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+use Illuminate\Support\Arr;
+
+class NewsController extends Controller
+{
+    protected $newsArr = [
+        0 => [
+            "images" => ['https://avatars.mds.yandex.net/get-zen_doc/1722013/pub_5e5f66317dd77672622a7756_5e5f6b8c5c0e620c44480247/scale_1200'], 
+            "title" => "Whales",
+            "newBody" => "Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam, sunt. Animi aliquam, voluptatum culpa neque nulla esse asperiores, quibusdam cumque repudiandae suscipit quos exercitationem harum tenetur sint! Tenetur maxime pariatur impedit et possimus maiores amet quo, debitis ullam perspiciatis, eius cumque libero corporis rem ducimus? Aliquid temporibus aspernatur harum autem?",
+            "hashtags" => ["animals", "nature", "whales"],
+            "date" => "12.01.2021",
+
+        ],
+        1 => [
+            "images" => ['https://pic.xenomorph.ru/2014-06/1402151154_f430_scuderia_spider_16m-01.jpg'], 
+            "title" => "Sportcar",
+            "newBody" => "Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea numquam aut eveniet, eaque neque reprehenderit blanditiis suscipit mollitia autem commodi eligendi nihil! Perspiciatis vero quo voluptates dignissimos veniam deleniti voluptate nisi assumenda nulla soluta consequuntur veritatis id eum error molestiae, maiores accusamus. Placeat assumenda deserunt fuga saepe culpa modi aspernatur sint reprehenderit labore laboriosam asperiores est autem debitis adipisci incidunt eos minus minima unde iure, at impedit blanditiis, amet quo recusandae? Consectetur illo officiis sunt maxime ipsam tenetur quibusdam adipisci in atque pariatur nam incidunt temporibus doloremque nesciunt provident dolor, debitis esse saepe, illum ullam eum quis asperiores consequatur consequuntur.",
+            "hashtags" => ["car", "road", "drive"],
+            "date" => "12.02.2021",
+
+        ],
+
+        2 => [
+            "images" => ['https://cherepah.ru/wp-content/uploads/3/8/8/3882c14363d51882ab239e032fde224a.jpeg'], 
+            "title" => "Tiger",
+            "newBody" => "Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam, sunt. Animi aliquam, voluptatum culpa neque nulla esse asperiores, quibusdam cumque repudiandae suscipit quos exercitationem harum tenetur sint! Tenetur maxime pariatur impedit et possimus maiores amet quo, debitis ullam perspiciatis, eius cumque libero corporis rem ducimus? Aliquid temporibus aspernatur harum autem?",
+            "hashtags" => ["animals", "nature", "tiger", "cats"],
+            "date" => "13.02.2021",
+
+        ],
+    ];
+
+    public function index(Request $request)
+    {
+        return view("home", [
```

Comment:
index like function of render, it sends default array in another function and it in turn in another, and each of those checking for some filteres and returns filtered array up